cmd/syncthing/main.go

```diff
@@ -220,8 +225,11 @@ The following are valid values for the STTRACE variable:
 // Environment options
 var (
 	noUpgradeFromEnv = os.Getenv("STNOUPGRADE") != ""
-	innerProcess     = os.Getenv("STNORESTART") != "" || os.Getenv("STMONITORED") != ""
+	isMonitored      = os.Getenv("STMONITORED") != ""
```

Comment:
What's the reason to add this?

Reply:
This is to distinguish between our monitor, and synctrayzor as a monitor, because synctrayzor might not put logs in a logfile like our monitor does, hence no `Open log` menu item.